jenkins_pipelines/environments/common/pipeline-build-validation-cleanup.groovy

```diff
@@ -138,7 +139,15 @@ def run(params) {
             stage('Delete client VMs') {
                 // Join the resources into a comma-separated string if there are any to delete
                 String tfResourcesToDeleteArg = params.tfResourcesToDelete ? '' : "--tf-resources-delete-all"
-
+                // WORKAROUND: Remove s390 clients manually until https://github.com/SUSE/spacewalk/issues/26502 is fixed.
+                sh """
+                    set +x
+                    source ~/.credentials
```

Comment:
Be caution here, as we have a set +x on the line above, it might be printing credentials visibile from the Jenkins web, that we might not want to show.